ui-tests/page-objects/meeting-input.js

```diff
@@ -0,0 +1,54 @@
+const PageObject = require('./page-object');
+
+const INPUT = '[data-qa-id=meeting-name-input]';
+const SUBMIT_BUTTON = '[data-qa-id=meeting-name-submit]';
+
+/**
+ * A page object for interacting with the input field for entering a meeting
+ * name to join.
+ */
+class MeetingInput extends PageObject {
+    /**
+     * Initializes a new {@code MeetingInput} instance.
+     *
+     * @param {Object} driver - The webdriver.io browser instance which can
+     * interact with Spot.
+     */
+    constructor(driver) {
+        super(driver);
+
+        this.rootSelector = INPUT;
+    }
+
+    /**
+     * Sets the input value.
+     *
+     * @returns {void}
```

Comment:
Isn't the param part missing? Or is it another ruleset that doesn't require having that?

Reply:
Looks like it has something to do with the eslint config having extended both plugin:react/recommended eslint-config-jitsi. I added the param. I'll try to find out why the problems occurs and fix the config and the subsequent errors in another PR.